corehq/apps/registration/forms.py

```diff
@@ -631,10 +619,17 @@ def clean(self):
 
             if prefixed_profile_key in custom_user_data:
                 profile_id = custom_user_data.pop(prefixed_profile_key)
-                self._validate_profile(profile_id)
```

Comment:
I don't think so because the validation is already defined on the profile field that's created in the `CustomDataEditor` form [here](https://github.com/dimagi/commcare-hq/blob/6fcaa9b6130afba415559877821d5074a312406a/corehq/apps/custom_data_fields/edit_entity.py#L225). That same field is inserted into `AdminInvitesUserForm` so the validation will be included.